src/devices/Mcp23xxx/Mcp230xx.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.I2c;
+
+namespace Iot.Device.Mcp23xxx
+{
+    public class Mcp230xx : Mcp23xxx
```

Comment:
Will these 2 new classes eventually become abstract classes as well in the future and have individual classes like Mcp23001.cs or mcp23S01.cs implement them?

Reply:
Updated to abstracts and added device classes.  Don't laugh, but I have a feeling I screwed this up 🤷‍♂️ 😄